src/App.vue

```diff
@@ -90,7 +94,9 @@ export default {
         this.$store.dispatch('updatePinnedItems'),
         this.$store.dispatch('updateUserProfile'),
         (async () => {
-          const balance = await this.sdk.balance(this.address).catch(() => 0);
+          const balance = await this.sdk.balance(this.address)
```

Comment:
`getBalance defaults to 0

Reply:
thanks, I remember that revert it on place, forgot about the second one

Reply:
@thepiwo it's already done, please check the next line with comment

Reply:
using `getBalance` defaults to 0, my comment doesn't seem to be resolved